.changeset/yellow-ladybugs-begin.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-tech-insights-backend': patch
+'@backstage/plugin-tech-insights-common': patch
```

Comment:
This really smells like it should be two different changesets right? Note that the text below will end up in the CHANGELOG.md of the respective packages, and make less sense in their isolated context.

Also, since recently, you actually should make a separate changeset for the new package as well. This is because we changed the release process for new packages to only get released when the rest of everything goes out on Thursdays, instead of immediately.